clippy_lints/src/methods/mod.rs

```diff
@@ -2590,11 +2590,11 @@ declare_clippy_lint! {
 declare_clippy_lint! {
     /// ### What it does
     /// Checks for usage of `x.get(0)` instead of
-    /// `x.first()`.
+    /// `x.first()` or `x.front()`.
     ///
     /// ### Why is this bad?
     /// Using `x.first()` is easier to read and has the same
-    /// result.
+    /// result, or `x.front()` for a `VecDeque`.
```

Comment:
How about "Using `x.first()` for `Vec`s and slices or `x.front()` for `VecDeque`s does the same and is easier to read."

Reply:
Yes, this looks better.